tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/ServiceRegistrations.cs

```diff
@@ -38,6 +38,14 @@ public static void RegisterCommonServices(IServiceCollection services)
             services.AddSingleton<ILanguageSpecificChecks, GoLanguageSpecificChecks>();
             services.AddSingleton<ILanguageSpecificCheckResolver, LanguageSpecificCheckResolver>();
 
+            // Update language services
+            // Register concrete update implementations so ActivatorUtilities can create them with DI
+            services.AddSingleton<Update.JavaUpdateLanguageService>();
```

Comment:
nit: Add `using Azure.Sdk.Tools.Cli.Services.Update` to avoid relative scope reference.